mailbox/scanning-search/src/test/java/org/apache/james/mailbox/store/search/SimpleMessageSearchIndexTest.java

```diff
@@ -238,17 +238,21 @@ public void multimailboxSearchShouldReturnUidOfMessageWithExpectedFromInAllMailb
 
     @Ignore
     @Override
-    public void searchWithFullTextShouldReturnMailsWhenHtmlBodyMatchesAndNonContinuousWords() throws Exception {
+    public void searchWithTextShouldReturnMailsWhenHtmlBodyMatchesAndNonContinuousWords() throws Exception {
     }
 
     @Ignore
     @Override
-    public void searchWithFullTextShouldReturnMailsWhenTextBodyWithExtraUnindexedWords() throws Exception {
+    public void searchWithTextShouldReturnMailsWhenTextBodyWithExtraUnindexedWords() throws Exception {
     }
 
     @Ignore
     @Override
-    public void searchWithFullTextShouldReturnMailsWhenHtmlBodyMatchesWithStemming() throws Exception {
+    public void searchWithTextShouldReturnMailsWhenHtmlBodyMatchesWithStemming() throws Exception {
     }
 
+    @Ignore
+    @Override
+    public void searchWithFulTextShouldReturnMailsWhenToAndBodyAndAttachmentMatches() throws Exception {
```

Comment:
FullText

Reply:
(no more needed, due to FullText not supported by this implementation)